site/en/tutorials/tuning_quickstart_python.ipynb

```diff
@@ -113,7 +113,7 @@
         "If you want to run this notebook in Colab start by uploading your\n",
```

Comment:
<p>Do we still need this if there's nothing being passed?</p><p> </p><p>I realise there's some pedagogical value in saying "hey, YSK about this because sometimes you'll need to set stuff" but it also doesn't do anything, so there's value in keeping it minimal too. I prefer to omit it myself, but don't have a strong opinion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>